admin_auto_filters/filters.py

```diff
@@ -48,9 +48,18 @@ def __init__(self, request, params, model, model_admin):
         remote_field = model._meta.get_field(self.field_name).remote_field
 
         attrs = self.widget_attrs.copy()
-       
+
+        field_desc = getattr(model, self.field_name)
+        try:
+            # First try to get the related using logic from ManyToManyDescriptor; any of these accesses might fail
+            related_model = field_desc.rel.related_model if field_desc.reverse else field_desc.rel.model
+            queryset = related_model.objects.get_queryset()
+        except:
```

Comment:
Lint: should use `except Exception` except in some exceptional cases.

Reply:
Or if the expected exception is AttributeError, best to catch only that.
Otherwise a simple typo causes a NameError that’s never seen and can take time to debug.

Reply:
Great call! Will take a look and experiment locally.